docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ## Catalog, Lambdas
 * [Added] Retry logic for failed queries, minimize load on ES for sample, images
+* [Added] Search help dropdown for the index landing page ([1838](https://github.com/quiltdata/quilt/pull/1838))
```

Comment:
i believe the pr number should be prefixed with a `#` by convention

Reply:
Thanks!